Islandora-CLAW/chullo
# Github actions

**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)


# What does this Pull Request do?
Converts to github actions
updates the badges
removes travis

# What's new?
* new CI runner

# How should this be tested?
* make sure the tests pass (i don't think they will until this PR is merged - but you can see them running over on the branch itself)
passed here: 
https://github.com/asulibraries/chullo/runs/2468272710?check_suite_focus=true
and codecov should be running as well (proof here https://codecov.io/github/asulibraries/chullo/commit/66f652f878ba617e56d5ffaf95ab02cadb39f031)




# Interested parties
@Islandora/8-x-committers
